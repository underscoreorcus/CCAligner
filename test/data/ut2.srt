1
00:00:01,500 --> 00:00:03,230
What's new at school?
Nothing.