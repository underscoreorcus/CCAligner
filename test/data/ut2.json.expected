{
	"subtitles": [
	{
		"subtitle" : "What's new at school? Nothing.",
		"edited_text" : "What's new at school Nothing",
		"start" : "1500",
		"end" : "3230",
		"words" : [
			{
				"word" : "What's",
				"recognised" : "1",
				"start" : "1560",
				"end" : "1810",
				"duration" : "250"
			},
			{
				"word" : "new",
				"recognised" : "1",
				"start" : "1820",
				"end" : "1960",
				"duration" : "140"
			},
			{
				"word" : "at",
				"recognised" : "1",
				"start" : "1970",
				"end" : "2070",
				"duration" : "100"
			},
			{
				"word" : "school",
				"recognised" : "1",
				"start" : "2080",
				"end" : "2770",
				"duration" : "690"
			},
			{
				"word" : "Nothing",
				"recognised" : "1",
				"start" : "2780",
				"end" : "3180",
				"duration" : "400"
			}
		],
		"phonemes" : [
		]
	}	]
}
